src/App.js

```diff
@@ -134,4 +143,15 @@ class App extends Component {
   }
 }
 
-export default App;
+function mapDispatchToProps(dispatch) {
+  return {
+    actions: bindActionCreators(actions, dispatch),
+  };
+}
+
+export default withRouter(
```

Comment:
Had to use **withRouter** as the routes stopped working after I used **connect**. https://github.com/ReactTraining/react-router/issues/4744